tardis/tests/tests_slow/test_w7.py

```diff
@@ -0,0 +1,165 @@
+import os
+import yaml
+import numpy as np
+import pytest
+from numpy.testing import assert_allclose
+from astropy import units as u
+
+from tardis.simulation.base import Simulation
+from tardis.model import Radial1DModel
+from tardis.io.config_reader import Configuration
+
+
+def data_path(fname):
+    return os.path.join(os.path.dirname(os.path.realpath(__file__)), "w7", fname)
+
+
+@pytest.mark.skipif(not pytest.config.getoption("--slow"),
+                    reason="slow tests can only be run using --slow")
+@pytest.mark.skipif(not pytest.config.getvalue("baseline-data"),
+                    reason="--baseline-data was not specified")
+@pytest.mark.skipif(not pytest.config.getvalue("atomic-dataset"),
+                    reason="--atomic-dataset was not specified")
+class TestW7(object):
+    """
+    Slow integration test for Stratified W7 setup.
+
+    Assumed two compressed binaries (.npz) are placed in `baseline/w7`
+    directory, whose path is provided by command line argument:
+
+    * ndarrays.npz                       | * quantities.npz
+    Contents (all (.npy)):               | Contents (all (.npy)):
+        * last_interaction_type          |     * t_rads
+        * last_line_interaction_out_id   |     * luminosity_inner
+        * last_line_interaction_in_id    |     * montecarlo_luminosity
+        * j_estimators                   |     * montecarlo_virtual_luminousity
+        * j_blue_estimators              |     * time_of_simulation
+        * last_line_interaction_shell_id |     * montecarlo_nu
+        * nubar_estimators               |     * last_line_interaction_angstrom
+        * ws                             |     * j_blues_norm_factor
+    """
+
+    @classmethod
+    @pytest.fixture(scope="class", autouse=True)
+    def setup(self):
+        """
+        This method does initial setup of creating configuration and performing
+        a single run of integration test.
+        """
+        self.config_file = data_path("config_w7.yml")
+        self.abundances = data_path("abundancies_w7.dat")
+        self.densities = data_path("densities_w7.dat")
+
+        # First we check whether atom data file exists at desired path.
+        self.atom_data_filename = os.path.expanduser(os.path.expandvars(
+                                    pytest.config.getvalue('atomic-dataset')))
```

Comment:
We may want to add a check that the atomic database is the same as used for the baseline data. Otherwise, this test most likely won't pass. Not sure if we have to include this check in the current PR, but something to keep in mind, though. Thoughts, @wkerzendorf, @karandesai-96, @yeganer?